session-replacement/src/main/java/com/amadeus/session/servlet/WebXmlParser.java

```diff
@@ -145,6 +145,10 @@ private static void lookForSessionConf(SessionConfiguration sessionConfiguration
     if (isNonEmpty(trackingMode)) {
       sessionConfiguration.setSessionTracking(sessionTracking(trackingMode));
     }
+    String path = xpath.evaluate("/web-app/session-config/cookie-config/path/text()", document);
+    if (isNonEmpty(path)) {
+      sessionConfiguration.setAttribute(CookieSessionTracking.COOKIE_PATH_PARAMETER, path);
```

Comment:
You can maybe COOKIE_CONTEXT_PATH_PARAMETER and change its semantic (from boolean to text)